web-ui/src/components/routes/NewUserRoutes.jsx

```diff
@@ -10,6 +10,7 @@ import WorkingLocationPage from "../../pages/WorkingLocationPage";
 import EquipmentPage from "../../pages/EquipmentPage";
```

Comment:
Lets get rid of any imports that are not being used so gradle will not fail any tests